config/voyager-forms.php

```diff
@@ -20,4 +20,8 @@
         'radio' => 'Radio',
         'file' => 'File'
     ],
+
+    'email' => [
```

Comment:
What does this do?

Reply:
$form->mailto is set as false instead of 'voyager.forms@mailinator.com' so that if the form does not have `mailto` field AND a default mailto email is not specified, it will skip sending the email

Reply:
Yep understood that, but what does this do? (config/voyager-forms.php > email['enabled'])? I can't see that you're using it anywhere

Reply:
I think I mistook your comment for other file.

Yes that line is not used anymore. I used it with the enabled setting before. I will remove it